code/modules/demo/hooks.dm

```diff
@@ -0,0 +1,24 @@
+/atom
```

Comment:
Nah, these overrides should be in their respective main files `atom.dm`, `mob.dm`, `client.dm`, etc...